src/sentry/static/sentry/app/components/events/interfaces/request.jsx

```diff
@@ -87,19 +95,23 @@ class RequestInterface extends React.Component {
 
     children.push(
       <h3 key="title">
-        <a href={isValidUrl ? fullUrl : null} title={fullUrl}>
+        <a href={fullUrl} title={fullUrl}>
```

Comment:
Hmm is this an external URL? Should we use `<ExternalLink>` so that it opens in a new tab?